src/i18n/en.json

```
@@ -524,12 +528,13 @@
         "language-es": "Spanish",
         "language-pt": "Portuguese",
         "language-hr": "Croatian",
-        "language-change-modal": "Changing the language will redirect you to another domain. Your settings and saved data from this domain will not carry over.<br>In particular, this means that things like your privacy settings and saved requests will not be available in the new language. You can still always go back to this language and view them here but you cannot share settings and saved data between different languages.",
-        "change-lang": "Change language",
-        "stay": "Stay here",
         "not-supported-prompt": "Your language or country is not supported yet?",
         "not-supported-cta": "Help us support it!"
     },
+    "interlingual": {
+        "recommend-en": "This site is also available in your langauge!",
+        "recommend-de": "Diese Seite ist auch in Ihrer Sprache verfügbar!"
```

```suggestion
        "recommend-de": "Diese Seite ist auch in Deiner Sprache verfügbar!"
```

We use the informal "you" :D

---

True - I think this however should now be moved to each of the language files in the requests section similar to how I'm doing it for the i18n element

---

Resolved in a0aa8fb